diff --git a/node_modules/@dcloudio/uni-h5/dist/uni-h5.es.js b/node_modules/@dcloudio/uni-h5/dist/uni-h5.es.js
index f58ad19..cfe875d 100644
--- a/node_modules/@dcloudio/uni-h5/dist/uni-h5.es.js
+++ b/node_modules/@dcloudio/uni-h5/dist/uni-h5.es.js
@@ -7042,6 +7042,9 @@ function useResizeSensorUpdate(rootRef, emit2, reset) {
 }
 function useResizeSensorReset(rootRef) {
   return () => {
+    if (!rootRef.value) {
+      return
+    }
     const {
       firstElementChild,
       lastElementChild
@@ -12812,7 +12815,7 @@ const nodeList2VNode = (scopeId, triggerItemClick, nodeList) => {
     }
     if (!hasOwn(node, "type") || node.type === "node") {
       let nodeProps = { [scopeId]: "" };
-      const tagName = node.name.toLowerCase();
+      const tagName = node.name?.toLowerCase();
       if (!hasOwn(TAGS, tagName)) {
         return;
       }
