diff --git a/node_modules/@dcloudio/uni-ui/lib/uni-list-chat/uni-list-chat.vue b/node_modules/@dcloudio/uni-ui/lib/uni-list-chat/uni-list-chat.vue
index 27d05dd..67b1bc5 100644
--- a/node_modules/@dcloudio/uni-ui/lib/uni-list-chat/uni-list-chat.vue
+++ b/node_modules/@dcloudio/uni-ui/lib/uni-list-chat/uni-list-chat.vue
@@ -427,7 +427,7 @@
 	}
 
 	.uni-list-chat__header-image {
-		margin: 1px;
+		margin: 0px;
 		/* #ifdef APP-NVUE */
 		width: 50px;
 		height: 50px;
