diff --git a/node_modules/@dcloudio/uni-h5/dist/uni-h5.es.js b/node_modules/@dcloudio/uni-h5/dist/uni-h5.es.js
index 4a9e6d2..c842b43 100644
--- a/node_modules/@dcloudio/uni-h5/dist/uni-h5.es.js
+++ b/node_modules/@dcloudio/uni-h5/dist/uni-h5.es.js
@@ -12908,7 +12908,7 @@ const nodeList2VNode = (scopeId, triggerItemClick, nodeList) => {
     }
     if (!hasOwn(node, "type") || node.type === "node") {
       let nodeProps = { [scopeId]: "" };
-      const tagName = node.name.toLowerCase();
+      const tagName = node.name?.toLowerCase();
       if (!hasOwn(TAGS, tagName)) {
         return;
       }
