diff --git a/node_modules/@dcloudio/uni-ui/lib/uni-list-chat/uni-list-chat.vue b/node_modules/@dcloudio/uni-ui/lib/uni-list-chat/uni-list-chat.vue
index d49fd7c..08fd277 100644
--- a/node_modules/@dcloudio/uni-ui/lib/uni-list-chat/uni-list-chat.vue
+++ b/node_modules/@dcloudio/uni-ui/lib/uni-list-chat/uni-list-chat.vue
@@ -275,7 +275,7 @@
 	$avatar-width: 45px;
 	$avatar-border-radius: 5px;
 	$avatar-border-color: #eee;
-	$avatar-border-width: 1px;
+	$avatar-border-width: 0px;
 	$title-size: 16px;
 	$title-color: #3b4144;
 	$title-weight: normal;
@@ -427,7 +427,7 @@
 	}
 
 	.uni-list-chat__header-image {
-		margin: 1px;
+		margin: 0px;
 		/* #ifdef APP-NVUE */
 		width: 50px;
 		height: 50px;
